src/pat/inject.js

```diff
@@ -112,6 +112,24 @@ define([
             _.execute(cfgs, $el);
         },
 
+        onButtonClick: function inject_onButtonClick(ev) {
+            ajax.onClickSubmit(ev); // make sure the submitting button is sent with the form
```

Comment:
I guess it's a matter of interpretation.  When I see `onClickSubmit` I think _when Submit is Clicked_, and **not** _if something is Clicked then Submit_.
Either way, a change like that would require simultaneously changing all calls to `onClickSubmit` in all existing patterns.  Out of the scope of this PR.